functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/configuration/SurefireForksConfigurationTest.java

```diff
@@ -66,15 +75,38 @@ private void verifyTestSuiteExecution(String... systemPropertiesPairs){
             .applyAsLocalChanges("Inlined variable in a method", "Adds new unit test", "fixes tests");
 
         // when
-        final List<TestResult> actualTestResults =
-            project
-                .build("config/impl-base")
-                .options()
-                    .withSystemProperties(systemPropertiesPairs)
-                    .configure()
-                .run();
+        ProjectBuilder projectBuilder = project.build();
+        final List<TestResult> actualTestResults = projectBuilder
+            .options()
+                .withSystemProperties(systemPropertiesPairs)
+                .withSystemProperties(ENABLE_REPORT_PROPERTY, "true")
+            .configure()
+            .run();
 
         // then
-        assertThat(actualTestResults).containsAll(expectedTestResults).hasSameSizeAs(expectedTestResults);
+        softly.assertThat(actualTestResults).containsAll(expectedTestResults).hasSameSizeAs(1);
```

Comment:
`softly.assertThat(actualTestResults).containsAll(expectedTestResults).hasSameSizeAs(1);` this will always break... :)

Reply:
dammit - forgot to change before pushed - I needed to break the test to try it locally. 
Sorry for it :-)

Reply:
No worries :)